project/MicroServiceBuild.scala

```diff
@@ -26,7 +23,8 @@ object MicroServiceBuild extends Build with MicroService {
     "uk.gov.hmrc" %% "hmrctest" % "2.3.0" % scope,
     "org.scalatest" %% "scalatest" % "2.2.6" % scope,
     "org.pegdown" % "pegdown" % "1.6.0" % scope,
-    "com.typesafe.play" %% "play-test" % PlayVersion.current % scope
+    "com.typesafe.play" %% "play-test" % PlayVersion.current % scope,
+    "com.github.tomakehurst" % "wiremock" % "2.5.1" % scope
```

Comment:
I think we need to agree on what mocking library to use across the projects - I've used ScalaMock in the frontend project but that can be refactored to use WireMock if it is better. It'll be nice if we had consistency  across the projects